src/commons/api/ca.rs

```diff
@@ -335,6 +335,18 @@ impl IssuedCert {
     pub fn replaces(&self) -> Option<&ReplacedObject> {
         self.replaces.as_ref()
     }
+
+    /// Returns a (possibly empty) set of remaining resources which
+    /// is the intersection of the encompassing resources and this
+    /// certificate's current resources. Returns None if the current
+    /// resource set is not overclaiming.
+    pub fn remaining_resources(&self, encompassing: &ResourceSet) -> Option<ResourceSet> {
```

Comment:
I have no idea if this is correct or not, I don't know the domain well enough.

Reply:
hopefully more clear now